packages/core/src/components/card/card.scss

```diff
@@ -78,3 +81,46 @@
     @include utilities.sr-only();
   }
 }
+
+.dso-card-heading a {
+  text-decoration: none;
+
+  dso-icon:last-child {
+    align-self: flex-start;
+  }
+}
+
+::slotted([slot="selectable"]) {
+  font-size: 0 !important;
+}
+
+::slotted([slot="heading"]) {
+  display: flex !important;
+  align-items: center !important;
+
+  margin-block-end: 0 !important;
+  margin-block-start: 0 !important;
+
+  font-size: 1.125em !important;
+  font-weight: 700 !important;
+  color: colors.$bosgroen !important;
+  line-height: 1.25 !important;
+}
+
+::slotted([slot="content"]) {
+  --_dt-rich-content-margin-block: #{units.$u1};
+  --_dt-rich-content-margin-block-end: 0;
+  --_dt-rich-content-margin-block-start: 0;
+}
+
+::slotted([slot="interactions"]),
+::slotted([slot="interaction"]) {
```

Comment:
`::slotted([slot="interaction"])` hoort bij Plekinfo Card en zal in deze stylesheet niets doen.

Reply:
verwijderd